src/theory/quantifiers/sygus/cegis_unif.cpp

```diff
@@ -207,6 +228,18 @@ bool CegisUnif::processConstructCandidates(const std::vector<Node>& enums,
     {
       d_sygus_unif.setConditions(
           e, cost_lit, unif_enums[1][e], unif_values[1][e]);
+      // if condition enumerator had value, exclude this value
+      if (options::sygusUnifCondIndependent() && !unif_enums[1][e].empty())
+      {
+        Node eu = unif_enums[1][e][0];
+        Assert(d_u_enum_manager.d_enum_to_active_guard.find(eu)
+               != d_u_enum_manager.d_enum_to_active_guard.end());
+        Node g = d_u_enum_manager.d_enum_to_active_guard[eu];
```

Comment:
Same thing here, make/use a function in EnumManager for getting the active guard for eu.